realm/realm-library/src/main/java/io/realm/internal/core/NativeMixed.java

```diff
@@ -73,7 +73,7 @@ public NativeMixed(ObjectId value) {
     }
 
     public NativeMixed(Decimal128 value) {
-        this(nativeCreateMixedDecimal128(value.getHigh(), value.getLow()));
+        this(nativeCreateMixedDecimal128(value.getLow(), value.getHigh()));
```

Comment:
Interesting. Was this just an error or has something changed....and are we sure that we fix it at all locations then ;)

Reply:
There were some pieces of code that were inverted. Now they should be aligned.